### PR TITLE
JIRA service desk api endpoint

### PR DESCRIPTION
**Pull request by lbiedinger**

### CODE REVIEW

File: src/server-middleware/jira-service-desk-api.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          requestTypeId: '81', // Type 81 is 'suggestion'
+          requestFieldValues: {
+            summary: parsedData.summary
+          }
+        };
+        if (parsedData.email) {
+          createRequestData.raiseOnBehalfOf = parsedData.email;
+        }
+
+        axios.post('https://europeana.atlassian.net/rest/servicedeskapi/request', createRequestData, {
```
Comment (lbiedinger) [inline src/server-middleware/jira-service-desk-api.js:25]: Should the endpoint could be passed via env var?

Comment (rwd) [reply src/server-middleware/jira-service-desk-api.js]: Yes, and is in my refactor.

File: src/server-middleware/api/index.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    runtimeConfig = defu(nuxtConfig.privateRuntimeConfig, nuxtConfig.publicRuntimeConfig);
+  }
+  next();
+});
+
+const debugMemoryUsage = require('./debug/memory-usage');
+app.get('/debug/memory-usage', debugMemoryUsage);
+
+const jiraServiceDesk = require('./jira/service-desk');
+app.post('/jira/service-desk', (res, req, next) => jiraServiceDesk(runtimeConfig.jira)(res, req, next));
```
Comment (lbiedinger) [inline src/server-middleware/api/index.js:21]: SonarCloud is raising a warning on the param order here.

Additionally, in fact I believe that we don't need to pass 'next' at all either.

Comment (rwd) [reply src/server-middleware/api/index.js]: I have addressed the security hotspot, which is about the use of `x-powered-by` header, and also removed `next` from the arguments.

**Review by mirjamverloop — APPROVED** [review]

**Pull request merged**